tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/SeleniumContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium.Support.UI;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Utils
+{
+    public class SeleniumContext : IDisposable
+    {
+        public IWebDriver WebDriver { get; }
+        public WebDriverWait WebWaiter{ get; }
+
+        public SeleniumContext()
+        {
+            ChromeOptions options = new ChromeOptions();
+            options.AddArguments("headless", "window-size=1920,1080", "no-sandbox", "disable-dev-shm-usage");
+            WebDriver = new RemoteWebDriver(new Uri("http://localhost:4444/wd/hub"), options);
```

Comment:
Might be worth adding in a way of running the tests locally (rather than via the grid) when debugging?